cmd/state/main.go

```diff
@@ -202,8 +209,8 @@ func run(args []string, isInteractive bool, out output.Outputer) error {
 	if childCmd == nil || !childCmd.SkipChecks() {
 		// Auto update to latest state tool version, only runs once per day
 		// Todo: This is better done in the `state-svc` process  https://www.pivotaltracker.com/story/show/177730748
-		if updated, err := autoUpdate(args, cfg, pjPath); err != nil || updated {
-			return err
+		if _, err := autoUpdate(args, cfg, pjPath); err != nil {
+			logging.Error("Failed to initialize auto update: %v", err)
 		}
```

Comment:
This is kind of unrelated.  But I ran into this reviewing the code: And I strongly believe (as we are now running this at the end of the State Tool), users might prefer this behavior.
Especially, when they are running automated commands, they probably do not want the State Tool to error out, because the auto-update could not be scheduled...